### PR TITLE
refactor: rename 'load' to 'occupancy'

### PR DESCRIPTION
**Pull request by woshilapin**
:mag: Please review commit by commit for your sanity

### CODE REVIEW

File: server/src/response.rs [hunk 1]
```diff
@@ -758,7 +758,7 @@ fn make_stop_datetimes(
         StopTimes::Base(_) => navitia_proto::RtLevel::BaseSchedule,
         StopTimes::New(_) => navitia_proto::RtLevel::Realtime,
     };
-    // `stop_time_idx` is only used for occupancy reporting (feature "vehicle_loads")
+    // `stop_time_idx` is only used for occupancy reporting (feature "vehicl_occupancy")
```
Comment (pbench) [inline server/src/response.rs:761]:
```suggestion
    // `stop_time_idx` is only used for occupancy reporting (feature "vehicle_occupancy")
```

File: src/occupancy_data/real_occupancy.rs [hunk 2]
```diff
@@ -175,31 +153,31 @@ impl Display for LoadsCount {
...
 }
 
-pub struct LoadsData {
-    per_vehicle_journey: BTreeMap<BaseVehicleJourneyIdx, VehicleJourneyLoads>,
+pub struct OccupancyData {
+    per_vehicle_journey: BTreeMap<BaseVehicleJourneyIdx, VehicleJourneyOccupancy>,
 }
 
-struct VehicleJourneyLoads {
+struct VehicleJourneyOccupancy {
```
Comment (pbench) [inline src/occupancy_data/real_occupancy.rs:171]:
```suggestion
struct VehicleJourneyOccupancies {
```
as it contains more than one occupancy data

File: src/occupancy_data/real_occupancy.rs [hunk 3]
```diff
@@ -175,31 +153,31 @@ impl Display for LoadsCount {
...
     stop_sequence_to_idx: BTreeMap<StopSequence, usize>,
-    per_date: BTreeMap<NaiveDate, TripLoads>,
+    per_date: BTreeMap<NaiveDate, TripOccupancy>,
 }
 
-struct TripLoads {
-    per_stop: Vec<Load>,
+struct TripOccupancy {
```
Comment (pbench) [inline src/occupancy_data/real_occupancy.rs:176]:
```suggestion
struct TripOccupancies {
```

File: src/occupancy_data/real_occupancy.rs [hunk 4]
```diff
@@ -215,54 +193,54 @@ impl VehicleJourneyLoads {
...
+                let trip_occupancy = vehicle_journey_occupancy.per_date.get(date)?;
+                let nb_of_stops = trip_occupancy.per_stop.len();
+                Some(&trip_occupancy.per_stop[..(nb_of_stops - 1)])
             }
             VehicleJourneyIdx::New(_) => None,
         }
     }
 
-    pub fn load(
+    pub fn occupancy(
```
Comment (pbench) [inline src/occupancy_data/real_occupancy.rs:221]:
```suggestion
    pub fn occupancies(
```

**Review by pbench — APPROVED** [review]

File: src/occupancy_data/real_occupancy.rs (hunk 3 above)
Comment (woshilapin) [reply src/occupancy_data/real_occupancy.rs]: Done in https://github.com/hove-io/loki/pull/371/commits/741b6171e542a78349fb496281e80332f408d052.

File: server/src/response.rs (hunk 1 above)
Comment (woshilapin) [reply server/src/response.rs]: Fixed in https://github.com/hove-io/loki/pull/371/commits/818cf091a50360a45d42b1843a6212ff74505cec. Thanks.

**Pull request merged**